libs/withdraws/src/lib/withdraw-manager.tsx

```diff
@@ -63,6 +65,10 @@ export const WithdrawManager = ({
     [accounts, assets, update, getThreshold, getDelay]
   );
 
+  if (assetId) {
```

Comment:
chore (blocking): state should not be updated like this in React

Reply:
i can set an initial state as in the comment below